setup/includes/upgrades/common/3.0.0-remove-tv-eval-system-setting.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Common upgrade script to clean up deprecated system settings for TV eval feature.
+ *
+ * @var modX
+ *
+ * @package setup
+ */
+
+$deprecatedSetting = $modx->getObject('modSystemSetting', array(
+    'key' => 'allow_tv_eval'
+));
+if ($deprecatedSetting instanceof modSystemSetting) {
+    if ($deprecatedSetting->remove()) {
+        $this->addResult(modInstallRunner::RESULT_SUCCESS,'<p class="ok">allow_tv_eval System Setting removed.</p>');
```

Comment:
Here should be `$this->runner->addResult` otherwise it produces errors about an undefined method of the class.
But I would prefer also refactor a bit `modInstallVersion` class to have method `getRunner` there.